Street Trees.xcodeproj/project.pbxproj

```diff
@@ -1438,6 +1449,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.CodeForOrlando.Street-Trees";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
+				SWIFT_VERSION = 2.3;
```

Comment:
Revert this to use swift 2.2. We do not support Xcode 8 or Swift 2.3 at this time.